### ISSUE TITLE
malware 

### ISSUE THREAD

**Issue by kirkczyler17** [Syntheticc/EXO-Stealer#6]
dont try to run it has a malware that can grabbed your account infos

**Comment by Syntheticc** [comment]
> dont try to run it has a malware that can grabbed your account infos

No! Really!! its not like it was made for people to use to grab info
